airbyte-integrations/connectors/source-jira/source_jira/streams.py

```diff
@@ -174,17 +174,17 @@ def path(self, **kwargs) -> str:
     def request_params(self, stream_state=None, **kwargs):
         stream_state = stream_state or {}
         params = super().request_params(stream_state=stream_state, **kwargs)
-        params["fields"] = ["attachment", "issuelinks", "security", "issuetype", "created"]
-        if stream_state.get(self.cursor_field):
-            issues_state = pendulum.parse(stream_state.get(self.cursor_field))
+        params["fields"] = ["attachment", "issuelinks", "security", "issuetype", "updated"]
+        if stream_state.get(self.cursor_field, None):
+            issues_state = pendulum.parse(stream_state[self.cursor_field])
             issues_state_row = issues_state.strftime("%Y/%m/%d %H:%M")
-            params["jql"] = f"created > '{issues_state_row}'"
+            params["jql"] = f"updated > '{issues_state_row}'"
         return params
 
     def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]) -> Mapping[str, Any]:
-        latest_record_date = pendulum.parse(latest_record.get("fields", {}).get(self.cursor_field))
+        latest_record_date = pendulum.parse(latest_record.get("fields", {}).get(self.cursor_field, ""))
```

Comment:
what if it is empty? also can it be empty?

Reply:
haven't seen such records and in the doc there is no information on this @keu

Reply:
so then, what your code is doing?